MobileCryptoService/MobileCryptoService/Presentation/HomeScreen/View/MarketView.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  MarketView.swift
+//  MobileCryptoService
+//
+//  Created by Dario Leonel Abatte on 07/11/2022.
+//
+
+import SwiftUI
+
+struct MarketView: View {
+    var body: some View {
+        ZStack{
+            Color("BackgroundColor")
+                .edgesIgnoringSafeArea(.all)
+            VStack {
+                HStack {
+                    Image("SmallProfilePhoto")
+                    Text("Good morning, \nJosé")
+                        .font(.custom("Mulish-Bold", size: 20))
+                    Spacer()
+                }
+                Text("Currency Selection")
+                    .font(.custom("Mulish-Bold", size: 20))
+                CryptoButtons(ButtonText1: "ARS", ButtonText2: "USD")
```

Comment:
Great We can improve at the future we can change to receive a list of items I will set up a little call to help you with that and teach you how improve it in order to generate a general component